File: Assignment_1/max_min.c
#include <stdio.h>
int main()
{
    int n;
    printf("Enter size of array:");
    scanf("%d",&n);
    int arr[n];
    for(int i=0;i<n;i++)
    {
        printf("Enter element:");
        scanf("%d",&arr[i]);
    }
    int max=arr[0], miin=arr[0];
    for(int i=0;i<n;i++)
    
    return 0;
}

File: Assignment_1/max_min.py
n=int(input("Enter size of array:"))
arr=[]
for i in range(n):
    arr.append(int(input("Enter element:")))
max=arr[0]
min=arr[0]
for item in arr:
    if item>max:
        max=item
    if item<min:
        min=item
print("Maximum element in array:",max)
print("Minimum element in array:",min)